.github/CODEOWNERS

```diff
@@ -56,6 +54,7 @@ build/ @janechu @nicholasrice @chrisdholt @awentzel
 /packages/utilities/fast-animation/ @nicholasrice @chrisdholt
 /packages/utilities/fast-colors/ @nicholasrice @chrisdholt
 /packages/utilities/fast-eslint-rules/ @nicholasrice @janechu @chrisdholt
+/packages/utilities/fast-tslint-rules/ @nicholasrice @janechu @chrisdholt
```

Comment:
This was purposely left out as this package will be removed in the coming weeks

Reply:
Thanks.  Removed